frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx

```diff
@@ -161,8 +161,7 @@ export default function MobileView(props: CodingViewProps) {
             />
           </ButtonRow>
 
-          {!!paid ? markUnpaid : null}
-
+          {!paid ? markUnpaid : markUnpaid}
```

Comment:
```suggestion
          { markPaidOrUnpaid }
```

Reply:
could you address this comment @gouravmpk

Reply:
Pushing